extensions/amp-story/1.0/story-analytics.js

```diff
@@ -132,7 +132,7 @@ export class StoryAnalyticsService {
     this.incrementPageEventCount_(eventType);
 
     getAmpdoc(this.element_)
-      .whenFirstVisible()
+      .whenNextVisible()
```

Comment:
is there any tests for this functionality?

Reply:
Yeah tried to have a small test too but this is also player-related complicated scenario, where you need to mimic unfocus of the tab. This similarly would be better tested with demo server.